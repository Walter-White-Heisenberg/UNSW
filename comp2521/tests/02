1057
1120
91
1923
161
1392
1364
1067
312
56
1513
770
1562
1842
853
200
919
1495
1567
1476
940
1756
1368
695
834
1373
91
75
1108
580
414
516
51
857
790
211
248
153
1277
911
209
1142
33
1770
983
885
1969
1901
731
1536
1376
23
1643
743
717
828
467
1159
902
1926
90
1668
441
140
524
1230
350
1123
1383
1979
385
1943
1120
417
1712
102
1301
32
354
32
1919
1729
406
1913
823
1474
740
1641
632
1994
1566
721
13
6
1212
888
1587
1561
10
1321
1891
746
1263
1010
1163
1326
1463
815
1358
1816
846
1628
1896
1603
1541
1070
1076
280
710
1707
625
275
779
637
632
1990
1524
571
1903
1885
1891
1793
631
1154
1155
1793
479
969
607
188
1137
1805
1816
1032
1759
1708
454
835
339
1163
893
964
1790
1672
1600
421
13
1476
991
1915
1360
1234
60
342
739
1566
486
1217
534
1445
1757
22
1601
1924
1406
1359
1631
1859
193
1969
1021
1086
932
1162
1109
884
1583
1121
711
925
1388
422
510
1799
764
1248
1364
1249
817
249
