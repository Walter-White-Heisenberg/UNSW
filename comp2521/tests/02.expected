6
10
13
13
22
23
32
32
33
51
56
60
75
90
91
91
102
140
153
161
188
193
200
209
211
248
249
275
280
312
339
342
350
354
385
406
414
417
421
422
441
454
467
479
486
510
516
524
534
571
580
607
625
631
632
632
637
695
710
711
717
721
731
739
740
743
746
764
770
779
790
815
817
823
828
834
835
846
853
857
884
885
888
893
902
911
919
925
932
940
964
969
983
991
1010
1021
1032
1057
1067
1070
1076
1086
1108
1109
1120
1120
1121
1123
1137
1142
1154
1155
1159
1162
1163
1163
1212
1217
1230
1234
1248
1249
1263
1277
1301
1321
1326
1358
1359
1360
1364
1364
1368
1373
1376
1383
1388
1392
1406
1445
1463
1474
1476
1476
1495
1513
1524
1536
1541
1561
1562
1566
1566
1567
1583
1587
1600
1601
1603
1628
1631
1641
1643
1668
1672
1707
1708
1712
1729
1756
1757
1759
1770
1790
1793
1793
1799
1805
1816
1816
1842
1859
1885
1891
1891
1896
1901
1903
1913
1915
1919
1923
1924
1926
1943
1969
1969
1979
1990
1994
